lib/kubot/db.rb

```diff
@@ -0,0 +1,112 @@
+require 'sqlite3'
+
+module Kubot
+  class KubotDB < SQLite3::Database 
+    def create_teams_table
+      self.execute <<-SQL
+    create table if not exists teams (
+    team_id varchar(30),
+    team_name varchar(30),
+    bot_id varchar(30),
+    access_token varchar(50),
+    bot_access_token varchar(50),
+    UNIQUE(team_id,access_token,bot_access_token)
+);
+      SQL
+    end
+
+    def add_team(team_id,team_name,bot_id,access_token,bot_access_token)  
+      self.execute("INSERT INTO teams (team_id, team_name,bot_id, access_token, bot_access_token)
+                   VALUES (?,?,?,?,?);",[team_id,team_name,bot_id,access_token,bot_access_token])
+    end
+
+    def create_channels_table
+      self.execute <<-SQL
+    create table if not exists channels (
+      support_channel varchar(30),
+      ask_channel varchar(30),
+      support_channel_bot_token varchar(50),
+      ask_channel_bot_token varchar(50),
+      UNIQUE(support_channel,ask_channel)
+    );
+      SQL
+    end
+
+    def add_channels(support_channel,ask_channel, support_channel_bot_token, ask_channel_bot_token)
+      self.execute("INSERT INTO channels (support_channel, ask_channel,support_channel_bot_token, ask_channel_bot_token)
+             VALUES (?,?,?,?)", [support_channel,ask_channel, support_channel_bot_token, ask_channel_bot_token])
+    end
+
+    def select_team_by_name(team_name)
+      self.execute("SELECT * FROM teams WHERE team_name = '#{team_name}'") do |row|
```

Comment:
there is sql injection, you should not pass variables like that, use the same scheme as you do in INSERT statements.